eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/trace/Trace.java

```diff
@@ -120,7 +120,7 @@ public Span addTraceInfoToSpan(ChannelHandlerContext ctx, CloudEvent cloudEvent)
 
         //add trace info
         for (String entry : cloudEvent.getExtensionNames()) {
-            span.setAttribute(entry, cloudEvent.getExtension(entry) == null ? "" : cloudEvent.getExtension(entry).toString());
+            span.setAttribute(entry, cloudEvent.getExtension(entry) == null ? "" : Object.requireNonNull(cloudEvent.getExtension(entry)).toString());
```

Comment:
this dependent on `java.util.Objects` please import, and change it to `Objects.requireNonNull`.